1-basics/index.html

```diff
@@ -1,10 +1,11 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8" />
+    <meta charset="utf-8"/>
     <title>React App</title>
   </head>
   <body>
     <div id="root"></div>
+    <script type="module" src="/src/main.jsx"></script>
```

Comment:
I will recommend to keep `index` so you can teach them that logic of having a default file.
Since they might join ongoing project in the future it's nice to make them familiar with that concept.
But feel free to decide what you prefer.

Reply:
I can rename `main` to `index` (`main` is the default name in vite templates, but I don't think it matters), however we still need to point to that file from the `index.html`. I think it's part of the CRA "magic" that it automatically detects your JS entry point, but I don't think it works in non-CRA contexts.

Reply:
Sure things, ignore it then